modules/apps/object/object-test/src/testFunctional/tests/ObjectFields.testcase

```diff
@@ -2106,6 +2106,93 @@ definition {
 			viewHomeFolder = "${siteName}");
 	}
 
+	@description = "LPS-156704 - Verify that the Excludes to operator is available when adding filters for the picklist fields to select data from the object"
+	@priority = "4"
+	test ExcludesIsAvailableForPicklistFields {
+		task ("Add a Custom Picklist and Item (via API)") {
+			Picklist.addPicklistViaAPI(picklistName = "Picklist Test");
+
+			Picklist.addPicklistItemViaAPI(
+				itemKey = "PicklistItemTest",
+				itemName = "Picklist Item Test",
+				picklistName = "Picklist Test");
+		}
+
+		task ("Add a Custom Object and Custom Picklist (via API)") {
+			ObjectAdmin.addObjectViaAPI(
+				labelName = "Custom Object 161899",
+				objectName = "CustomObject161899",
+				pluralLabelName = "Custom Objects 161899");
```

Comment:
Use the ticket number correct to add the Custom Object - [LPS-165981](https://issues.liferay.com/browse/LPS-165981)